cli/api/entity_config/config.go

```diff
@@ -52,7 +52,7 @@ func ConfigValueAsBytes(network *net.Network, application, entity, config string
 
 func SetConfig(network *net.Network, application, entity, config, value string) (string, error) {
 	url := fmt.Sprintf("/v1/applications/%s/entities/%s/config/%s", application, entity, config)
-	val := []byte(value)
+	val := []byte("\""+value+"\"")
```

Comment:
I'd say we do care about backwards compatibility.  I'll reverse my earlier suggestion about defaults. Will update the comment...